src/google-classroom-extractor/tests/mapping/user_submission_activities/test_user_submission_activities.py

```diff
@@ -129,7 +135,7 @@ def it_should_have_correct_shape(assignment_submissions_dicts):
         row_count, column_count = submissions_df.shape
 
         assert row_count == 2
-        assert column_count == 11
+        assert column_count == 13
```

Comment:
Also need to assert new state_row and grade_row values specifically.

Reply:
✔️